src/index.ts

```diff
@@ -141,6 +167,7 @@ export function HTPipe(): {};
 // one parameter - returns a new object with all the valid lifecycle stages of the parameter
 export function HTPipe<
   T extends OptionallyHasInitPreContext<any, any> &
+    OptionallyHasSanitizeParams<any, any> &
     OptionallyHasAttachData<any, any>
```

Comment:
- [ ] you forgot to add 'sanitizeParams' to the pick list below

Reply:
done